evaluation/fuzz/check.sh

```diff
@@ -19,7 +24,7 @@ width=$"$dir/width"
 
 ./"$script_dir/gen_float.py" gen --width $(cat $width) "$count" > "$data"
 
-(fud e -s cocotb.data "$data" --to cocotb-out "$dir/harness.fil" -s calyx.flags ' -d canonicalize' -q | \
+(fud e -s cocotb.data "$data" --to cocotb-out "$dir/harness.fil" -s calyx.flags ' -d canonicalize' -s filament.flags " $fil_flags"  -q | \
```

Comment:
LGTM! Can you coordinate with @gabizon103 and update the mono PR to use this once merged?